src/functions-and-arrays.js

```diff
@@ -1,19 +1,76 @@
 // Iteration #1: Find the maximum
 
+const maxOfTwoNumbers = (number1, number2) => {
+  if (number1 > number2) {
+    return number1;
+  } else {
+    return number2;
+  }
+  };
+
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+/* Declare a function named `findLongestWord` that takes as an argument an array of words
+and returns the longest one. If there are 2 with the same length,
+it should return the first occurrence. */
+
+const words = ['mystery', 'brother', 'aviator', 'crocodile', 'icecreama', 'pearl', 'orchard', 'crackpot'];
+
+function findLongestWord(arr) {
+  let longestWord = "";
+  if (arr === null || arr.length === 0) {
+    return null;
+  }
+  for (var i = 0; i < arr.length; i++){
+    if(arr[i].length > longestWord.length){
+        longestWord = arr[i];
+    }
+  }
+  return longestWord;
+}
+
+
 
 // Iteration #3: Calculate the sum
 
-const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+//const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+ 
+function sumNumbers(arr) {
+  let sum = 0;
+  for (let i = 0; i < arr.length; i++){
+      sum = arr[i] + sum;
```

Comment:
Good work Sabela! 

You can also write  use `sum += arr[i]` instead of `sum = arr[i] + sum`